gulpfile.js

```diff
@@ -88,7 +88,7 @@ gulp.task('compile', () => {
 });
 
 gulp.task('test.system.copy', () => {
-  return gulp.src(['system-test/**/*'])
+  return gulp.src(['system-test/**/*.js'])
```

Comment:
I think we may need to copy all extensions excluding `ts`. There are some `json` files for now I think and we may have other fixture type files in the future.